### PR TITLE
Phase2-hgx106 Step toward HGCal interface of Geometry

### PR DESCRIPTION
**Pull request by bsunanda**
Defines all parameters needed to for deciding numbering cells

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by civanch** [comment]
+1

**Comment by bsunanda** [comment]
@kpedro88 Can you please approve this?

**Comment by kpedro88** [comment]
+1

**Comment by bsunanda** [comment]
@ianna Please review and approve this

File: Geometry/HGCalCommonData/interface/HGCalParametersFromDD.h [hunk 1]
```diff
@@ -13,6 +14,9 @@ class HGCalParametersFromDD {
 
   bool build(const DDCompactView*,  HGCalParameters&, const std::string&,
 	     const std::string&, const std::string&);
+
+private:
+  double getDDDValue(const char* s, const DDsvalues_type& sv);
```
Comment (ianna) [inline Geometry/HGCalCommonData/interface/HGCalParametersFromDD.h]: @bsunanda - I do not see you using this here. Please remove it.

**Review by ianna — CHANGES_REQUESTED** [review]

**Comment by ianna** [comment]
+1

**Comment by fabiocos** [comment]
+1

**Pull request merged**